source/_integrations/homematic.markdown

```diff
@@ -459,7 +459,19 @@ binary_sensor:
           - sensor.time
         value_template: >-
           {{ as_timestamp(now()) - as_timestamp(state_attr('sensor.office_voltage', 'last_changed')) < 600 }}
+```
+
+Using new template sensor (prefered):
+```yaml
+template:
+  - sensor:
+      - name: "Homematic is sending updates"
+        state: >-
+          {{ as_timestamp(now()) - as_timestamp(states.sensor.outlet_washmachine_voltage.last_changed) < 600 }}
```

Comment:
Never access states via the `states` object directly.

See also:

![image](https://user-images.githubusercontent.com/195327/128700635-6a4cc0fc-21bc-41d0-b69e-f9739537d525.png)

https://www.home-assistant.io/docs/configuration/templating/#states